doc/contributing/docs/markup/_includes/file.rst

```diff
@@ -0,0 +1,2 @@
+:file:`/usr/example.py`
+:file:`/usr/{filename}/example.py`
```

Comment:
it's actually a `dirname` here :)

Let's give an extra example with filename, like this:
```suggestion
:file:`/usr/bin/example.py
:file:`/usr/{dirname}/example.py`
:file:`/usr/{dirname}/{filename.ext}`
```